src/components/front-pages/schedules.vue

```diff
@@ -262,6 +276,8 @@ export default {
     }
   },
   created () {
+    this.getActiveYear()
+    this.getYears()
```

Comment:
Should also be in vuex. I can help you work it in. 

Also should be called as app loads so that it’s ready to use by time pages load.